partitioned-table.md

```diff
@@ -164,6 +164,111 @@ Range partitioning is particularly useful when one or more of the following cond
 * You want to use a column that contains time or date values, or containing values arising from some other series.
 * You need to frequently run queries on the columns used for partitioning. For example, when executing a query like `EXPLAIN SELECT COUNT(*) FROM employees WHERE separated BETWEEN '2000-01-01' AND '2000-12-31' GROUP BY store_id;`, TiDB can quickly know that only the data in the `p2` partition needs to be scanned, because the other partitions do not match the `WHERE` condition.
 
+### Range INTERVAL partitioning
+
+INTERVAL partitioning is introduced in TiDB v6.3.0 as syntactic sugar:
+
```

Comment:
To let users know better about this feature, could we also add a sentence to describe what Range INTERVAL partitioning is?

For example:

Range INTERVAL partitioning is an extension of Range partitioning, which allows you to create partitions of a specified interval. Starting from TiDB v6.3.0, INTERVAL partitioning is introduced as syntactic sugar:

Reply:
Good idea. Updated.